mmv1/products/vertexai/EndpointWithModelGardenDeployment.yaml

```diff
@@ -852,15 +852,11 @@ properties:
         description: |-
           The user-specified display name of the endpoint. If not set, a
           default name will be used.
-      - name: dedicatedEndpointEnabled
```

Comment:
Currently, it change renames the `dedicatedEndpointEnabled` field to `dedicatedEndpointDisabled`. This is a breaking change for the Terraform provider. To handle this according to our standard practice, please follow the following steps:
- add the new field `dedicatedEndpointDisabled`
- mark the existing `dedicatedEndpointEnabled` deprecated by setting the `deprecation_message` on it and suggesting the use of the new field.  https://googlecloudplatform.github.io/magic-modules/breaking-changes/make-a-breaking-change/#field-deprecation-due-to-removal-or-rename
- add test coverage for the new field. https://googlecloudplatform.github.io/magic-modules/test/test/#add-resource-tests